loader/loader.go

```diff
@@ -443,3 +444,18 @@ func (l *loader) GetAutoReleasePlansForApplication(c client.Client, ctx context.
 
 	return &releasePlans.Items, nil
 }
+
+// GetScenario returns integration test scenario requested by name and namespace
+func (l *loader) GetScenario(c client.Client, ctx context.Context, name, namespace string) (*v1beta1.IntegrationTestScenario, error) {
+	scenario := &v1beta1.IntegrationTestScenario{}
```

Comment:
We may replace these code in this function with the code like [this](https://github.com/redhat-appstudio/integration-service/blob/main/loader/loader.go#L118-L119) 
```
	scenario := &v1beta1.IntegrationTestScenario{}
	return scenario, toolkit.GetObject(name, namespace, c, ctx, scenario)
```